<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationWindow</class>
 <widget class="QDialog" name="VisualizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization of genotypes</string>
  </property>
  <layout class="QVBoxLayout" name="visualizationVLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Visualization of genotypes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;The probabilities to be homozygous / heterozygous, at the QTL positions, have been computed according to favourable / unfavourable grouping of founder alleles.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a name=&quot;result_box&quot;&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;S&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;et a threshold and select a color to display a new view of the molecular score table based on genotypes.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:6pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Customize cut-off colors:</string>
     </property>
     <layout class="QVBoxLayout" name="groupBoxVLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="homoFavHLayout">
        <item>
         <widget class="QLabel" name="homoFavLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Probability to be homozygous favorable at the QTL position</string>
          </property>
          <property name="text">
           <string>Prob(+/+)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="homoFavSupLabel">
          <property name="text">
           <string> ≥ </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="homoFavDoubleSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>29</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.501000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.750000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="homoFavHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="homoFavButton">
          <property name="minimumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="heteroHLayout">
        <item>
         <widget class="QLabel" name="heteroLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Probability to be heterozygous at the QTL position</string>
          </property>
          <property name="text">
           <string>Prob(+/-)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="heteroSupLabel">
          <property name="text">
           <string> ≥ </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="heteroDoubleSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>29</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.501000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.750000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="heteroHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="heteroButton">
          <property name="minimumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="homoUnfavHLayout">
        <item>
         <widget class="QLabel" name="homoUnfavLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Probability to be homozygous unfavorable at the QTL position</string>
          </property>
          <property name="text">
           <string>Prob(-/-)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="homoUnfavSupLabel">
          <property name="text">
           <string> ≥ </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="homoUnfavDoubleSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>29</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.501000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.750000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="homoUnfavHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="homoUnfavButton">
          <property name="minimumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="questionTagHLayout">
        <item>
         <widget class="QLabel" name="questionTagLabel">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Uncertain genotype at the QTL position</string>
          </property>
          <property name="text">
           <string>The rest: uncertain genotypes (?)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="questionTagHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="questionTagButton">
          <property name="minimumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>46</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VisualizationWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VisualizationWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
